drivers/clock_control/clock_control_nrf2_hsfll.c

```diff
@@ -167,6 +179,28 @@ static int api_get_rate_hsfll(const struct device *dev,
 
 static int hsfll_init(const struct device *dev)
 {
+	const struct hsfll_dev_config *dev_config = dev->config;
+	nrf_hsfll_trim_t trim = {
+		.vsup   = sys_read32(dev_config->trim_vsup),
+		.coarse = sys_read32(dev_config->trim_coarse),
+		.fine   = sys_read32(dev_config->trim_fine),
```

Comment:
It looks like the `TCOEF` trim is missing from the list.

Reply:
Yes, it's missing. The nrfx HAL needs to be extended in this regard as currently the `nrf_hsfll_trim_t` structure does not contain the related field.

Reply:
Do you plan to fix this in this PR?